examples/input-element-default-checkedness/README.md

```diff
@@ -0,0 +1,38 @@
+# input-element-default-checkedness
```

Comment:
I didn't read the paragraphs closely but I scanned the structure/titles and it seems like you're hitting on everything important.

So, if I have any minor feedback about phrasing I'm sure I'll just PR it whenever I eventually need to touch this file for some reason.